plugin-server/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Any idea why this wobbles sometimes?

Reply:
None 🙈 

I assume some kind of global node and pnpm verson conflict

Reply:
what's your pnpm version? 🤔 I'm on 8.6.1 and node 18.16 (and it always gives me 6.1)

Reply:
So my higher version of pnpm gave me a lower version of lockfile 🙃

Reply:
whaaaaa

Reply:
downgraded to both 8.6.1 and 8.6.2 and running `pnpm i --force` switch lockfile version to 6.1

🙃